infra/bootstrap/account/main.tf

```diff
@@ -0,0 +1,59 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  # Profile that will be used to select which account to deploy to. This profile name should be propagated to the
+  # environment(s) that will share this aws account.
+  profile = "default"
+  # Choose the region where this infrastructure should be deployed.
+  region = "us-east-1"
+  # Set project tags that will be used to tag all resources. 
+  tags = merge(module.common.default_tags, {
+    description = "Backend resources required for terraform state management."
+
+  })
```

Comment:
My thought here was that since this is not technically an environment, and could contain multiple environments it could cause confusion if I pre-named it as such. Resources will still get tagged with what environment they belong to in the envs/env main.tf